python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -0,0 +1,159 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is adapted from PyTorch Lightning Tutorial.
+# https://github.com/PyTorchLightning/lightning-tutorials/blob/main/
+# lightning_examples/cifar10-baseline/baseline.py
+
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+from pytorch_lightning import LightningModule, Trainer, seed_everything
+from pytorch_lightning.callbacks import LearningRateMonitor
+from pytorch_lightning.loggers import TensorBoardLogger
+from torch.optim.lr_scheduler import OneCycleLR
+from torchmetrics.functional import accuracy
+
+try:
+    from pl_bolts.datamodules import CIFAR10DataModule
+    from pl_bolts.transforms.dataset_normalizations import cifar10_normalization
+except ModuleNotFoundError:
+    raise ImportError("This example need lightning-bolts, please install by: "
```

Comment:
I think you can just add installing lightning-bolts in this examples' readme file and not modify the code.

Reply:
Modified.